src/app/layout/redirect-status/redirect-status.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+import { RedirectData } from '../../models/redirect-data';
+
+@Component({
+  selector: 'f8-redirect-status',
+  templateUrl: 'redirect-status.component.html'
+})
+export class RedirectStatusComponent implements OnInit {
+
+  private redirects: Map<string, any> = new Map([
```

Comment:
I'd still like to see the `any` here replaced with a type definition describing the shape of the success/fail RedirectData structure within.

Reply:
Updated type. Is it better now?